docs/_docs/Developers/build.md

```diff
@@ -5,45 +5,91 @@ order: 1
 toc: true
 ---
 
-## Ubuntu 18.04/18.10 and Linux Mint 19
+## Ubuntu 18.04/18.10
+*Updated by [kreyren](https://github.com/Kreyren) at 24.02.2019*
```

Comment:
This is part of the git commit history and doesn't belong here.

Reply:
@plata Elaborate? Meaning excluding https://github.com/PhoenicisOrg/phoenicis/pull/1860/commits/2e5b87df599442c84c63d2cd357295c3ded89121 ?

Reply:
When we merge this PR your name, the date and the changes you've done are automatically logged by git and therefore don't need to be added separately to the file.

Reply:
Fair enough

Reply:
Fixed in https://github.com/PhoenicisOrg/phoenicis/pull/1860/commits/309b336e4bb0dd9007704d52b47268f4137cc10e